tests/php/src/Proxies/LegacyProxyTest.php

```diff
@@ -33,7 +33,7 @@ public function setUp() {
 	 */
 	public function test_get_instance_of_throws_when_trying_to_get_a_namespaced_class() {
 		$this->expectException( \Exception::class );
-		$this->expectExceptionMessage( 'The LegacyProxy class is not intended for getting instances of classes in the src directory, please use constructor injection or the instance of \Psr\Container\ContainerInterface for that.' );
+		$this->expectExceptionMessage( 'The LegacyProxy class is not intended for getting instances of classes in the src directory, please use method injection or the instance of \Psr\Container\ContainerInterface for that.' );
```

Comment:
Since this error message is intended to be "educational", maybe we could explicitly include the name of the method?